services/sentry/lib/eventReducer.js

```diff
@@ -73,7 +74,7 @@ function addAndToString(first, second) {
 }
 
 function isEmpty(aggr) {
-	return aggr.earners.length === 0
+	return aggr.earners.length === 0 && !aggr.events[eventTypes.close]
```

Comment:
Also instead of doing this why not just add the `creator` to the earner's array i.e.
```
aggr.earners.push(creator)
```

Reply:
https://github.com/AdExNetwork/adex-validator/blob/bebf3e295af56022c280d118bce44eccc7af4dc1/services/sentry/lib/eventReducer.js#L19